src/VirtualClient/VirtualClient.Main/profiles/PERF-MEM-LMBENCH.json

```diff
@@ -1,42 +1,55 @@
 {
-    "Description": "LMbench Performance Workload",
-    "MinimumExecutionInterval": "00:10:00",
-    "Metadata": {
-        "RecommendedMinimumExecutionTime": "(4-cores)=04:00:00,(16-cores)=10:00:00,(64-cores)=16:00:00",
-        "SupportedPlatforms": "linux-x64,linux-arm64",
-        "SupportedOperatingSystems": "CBL-Mariner,CentOS,Debian,RedHat,Suse,Ubuntu"
+  "Description": "LMbench Performance Workload",
+  "MinimumExecutionInterval": "00:10:00",
+  "Metadata": {
+    "RecommendedMinimumExecutionTime": "(4-cores)=04:00:00,(16-cores)=10:00:00,(64-cores)=16:00:00",
+    "SupportedPlatforms": "linux-x64,linux-arm64",
+    "SupportedOperatingSystems": "CBL-Mariner,CentOS,Debian,RedHat,Suse,Ubuntu"
+  },
+  "Parameters": {
+    "CompilerName": "gcc",
+    "CompilerVersion": "10",
+    "CompilerFlags": "CPPFLAGS=\"-I /usr/include/tirpc\"",
```

Comment:
Could you document the implication of the flags and what are the perf impact on the non-ubuntu22 systems?

Reply:
In some systems after installing the required packages few links for the compiler are by default set as part of installation process. But on ubuntu 22 systems, these links to the compiler are not set at the time of installation and needed to be explicitly passed as flags to the compiler to set up the links. 
Compilation itself throws error displaying required files are not found. 
fatal error: rpc/rpc.h: No such file or directory. These files/libraries are used as part of lmbench compilation.